.github/workflows/cicd.yml

```diff
@@ -0,0 +1,97 @@
+name: cicd
+
+on:
+  push:
+    branches: ['main', 'release', 'development', 'feature/**'] # '**' match any character including '/'. '*' match any character excluding '/'.
+  pull_request:
+    branches: ['main', 'release', 'development']
+
+env:
+  # if it is a main, release, or development branch, we want to create and push docker image
+  # DOCKERIZE: ${{ endsWith(github.ref, '/main') || endsWith(github.ref, '/release') || endsWith(github.ref, '/development') }}
+  # DOCKERIZE: ${{ (github.ref == 'refs/heads/main') || (github.ref == 'refs/heads/release') || (github.ref == 'refs/heads/development') }}
+  DOCKERIZE: 'false' # endsWith(github.ref, '/main') # temporarily I let's just create docker image only for main build
```

Comment:
Temporarily setting this to false. Still looking for options to push docker images.